src/planning.py

```diff
@@ -240,6 +251,8 @@ def _skeleton_generator(
     rng_prio = np.random.default_rng(seed)
     hq.heappush(queue,
                 (heuristic(root_node.atoms), rng_prio.uniform(), root_node))
+    visited_skeletons: Set[Tuple[_GroundNSRT, ...]] = set(tuple())  #initialize
```

Comment:
in general, please make a space between the hashtag and the first word in your comments

Reply:
also, since this is split over two lines, let's just make it a comment above. so:
```
# Initialize with empty skeleton for root.
visited_skeletons: Set[Tuple[_GroundNSRT, ...]] = set(tuple())
```

Reply:
also, can you add a comment explaining why we need to keep track of visited skeletons now? (this will also be a good opportunity to check your understanding!)